src/Jira/Api/Result.php

```diff
@@ -78,6 +78,17 @@ public function getIssues()
 
         return array();
     }
+    
+    public function getErrors()
+    {
+    	if (isset($this->result['errors'])) {
```

Comment:
What would make sense instead is to just return value of `$this->result['errors']` as is, when it's set.